tests/e2e/specs/shopper/cart-update-product-quantity.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { SIMPLE_PRODUCT_NAME } from '../../../utils/constants';
+
+const block = {
+	name: 'Cart',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Cart → Can update product quantity', () => {
+	beforeAll( async () => {
+		await shopper.block.emptyCart();
+	} );
+
+	afterAll( async () => {
+		await shopper.block.emptyCart();
+	} );
+
+	it( 'allows customer to update product quantity via the input field', async () => {
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
+		await shopper.block.goToCart();
+
+		await shopper.block.setCartQuantity( SIMPLE_PRODUCT_NAME, 4 );
+		await expect( page ).toMatchElement(
+			'button.wc-block-cart__submit-button[disabled]'
+		);
+
+		// To avoid flakiness: The default "idleTime: 500" fails in headless mode
+		await page.waitForNetworkIdle( { idleTime: 1000 } );
```

Comment:
I think you can use the `uiUnblocked()` function from the [woo e2e-utils](https://github.com/woocommerce/woocommerce/blob/5af5be510032f9de1b1c6c6dacbbe24e53f8b691/packages/js/e2e-utils/src/page-utils.js#L95) here. I'm not sure it works with the block, but it's the standard way of waiting for UI updates in Woo